validator/keymanager/v2/direct/direct.go

```diff
@@ -303,7 +303,6 @@ func (dr *Keymanager) initializeSecretKeysCache(ctx context.Context) error {
 	if len(accountNames) == 0 {
 		return nil
 	}
-	log.Info("Decrypting validator accounts...")
```

Comment:
I removed this because it wasn't always immediately above the progress bar so I was wondering "What is the progress bar doing?" well it's loading accounts so I added that as a progress bar description. See PR description for example output.